_posts/2023-11-16-virginia-beach-city/virginia-beach-city.Rmd

```diff
@@ -0,0 +1,229 @@
+
+::: {style="display: flex;"}
+::: {style="flex-direction: column; margin-right: 5px;"}
+![Figure 1: *Virginia Beach City Council Districts*](citycouncil.svg){width="328"}
+:::
+
+::: {style="flex-direction: column; margin-right: 5px;"}
+![Figure 2: *Majority Minority Districts in Virginia Beach City*](citycouncilmmd.svg){width="328"}
+:::
+
+::: {style="flex-direction: column;"}
+![Figure 3: *Partisan Breakdown of Districts in Virginia Beach City*](citycouncilparty.svg){width="328"}
+:::
+:::
+
+The eleventh member is elected at-large to serve as the mayor of the city. While the council is legally nonpartisan, candidates with political affiliations emerged just seven years after the formation of the city council in 1963. Historical elections have been competitive between both Democrats and Republicans with a few declared independents, but the overall majority of the city council historically leaned Republican, especially prior to the 2022 court-ordered redistricting. The current 2023 City Council is comprised of six Democrats and five Republicans (Figure 3), with Republican Bobby Dyer serving as mayor. No candidates were up for reelection in the November 7, 2023 Virginia general election.
+
+| [Mayor]{.smallcaps} | [1]{.smallcaps}   | [2]{.smallcaps}       | [3]{.smallcaps}       | [*4*]{.smallcaps}         | [5]{.smallcaps}     | [6]{.smallcaps}  | [*7*]{.smallcaps}    | [8]{.smallcaps}  | 9                  | *10*                 |
+|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|
+| Bobby Dyer (R)      | Rocky Holcomb (R) | Barbara M. Henley (D) | Michael Berlucchi (R) | *Amelia Ross-Hammond (D)* | Rosemary Wilson (R) | Worth Remick (D) | *Sabrina Wooten (D)* | Chris Taylor (R) | Joash Schulman (D) | *Jennifer Rouse (D)* |
+
+: Table 1: Current Virginia Beach City Council makeup. Majority-Minority Districts are italicized.
+
+The ten districts are constructed from 108 voting precincts displayed in Figure 1 below. Census blocks, which are areas bounded by visible features such as roads or streams and invisible features such as property lines or other city/school districts, are visualized in Figure 2. These blocks help visualize where the city is located, as census blocks often follow city blocks in urban environments.
+
+::: {style="display: flex;"}
+::: {style="flex-direction: column; margin-right: 10px;"}
+![Figure 4: *Map of Voting Precincts and City Council Boundaries in Virginia Beach City*](fig1ccandprecincts.png){width="328"}
+:::
+
+::: {style="flex-direction: column;"}
+![Figure 5: *Map of Census Blocks and City Council Boundaries in Virginia Beach City*](fig2ccandblocks.png){width="328"}
```

Comment:
What is the larger boundary here that contains the subset of the city council districts? (let me know if this question doesn't make sense)

Reply:
I'm a bit confused- do you mean the larger block that extends past the city on the north and east? If so, I think the census is just required to assign a block for the associated oceanfront administered by the city